src/domain/profile/children/ProfileChildrenList.tsx

```diff
@@ -88,15 +86,12 @@ const ProfileChildrenList: FunctionComponent = () => {
         )}
       </div>
       <Button
+        variant={'secondary'}
         aria-label={t('child.form.modal.add.label')}
         className={styles.addChild}
+        iconLeft={<IconPlusCircle />}
         onClick={() => setIsOpen(true)}
       >
-        <Icon
-          src={addIcon}
-          alt={t('child.form.modal.add.label')}
-          className={styles.addChildIcon}
-        />
         <span>{t('child.form.modal.add.label')}</span>
```

Comment:
Do you still need this span?

Reply:
Nice find. Was able to remove some css imports and variables!